ts/features/it-wallet/screens/ItwCredentialDetails.tsx

```diff
@@ -67,6 +74,53 @@ const ItwCredentialDetails = () => {
     </ScrollView>
   );
 
+  /**
+   * Renders the error view.
+   */
+  const ErrorView = (error: ItWalletError) => {
+    const mappedError = mapRequirementsError(error);
+    const cancelButtonProps = {
+      block: true,
+      light: false,
+      bordered: true,
+      onPress: navigation.goBack,
+      title: I18n.t("features.itWallet.generic.close")
+    };
+    return (
+      <>
+        <InfoScreenComponent
+          title={mappedError.title}
+          body={mappedError.body}
+          image={<Pictogram name="error" />}
+        />
+        <FooterWithButtons
+          type={"SingleButton"}
+          leftButton={cancelButtonProps}
+        />
+      </>
+    );
+  };
+
+  const RenderMask = () =>
+    pot.fold(
+      decodedPid,
+      () => <LoadingView />,
+      () => <LoadingView />,
+      () => <LoadingView />,
+      err => ErrorView(err),
+      some =>
+        pipe(
+          some.decodedPid,
+          O.fold(
+            () => <> </>, // TODO: https://pagopa.atlassian.net/browse/SIW-364
+            decodedPid => <ContentView decodedPid={decodedPid} />
+          )
+        ),
+      () => <LoadingView />,
+      () => <LoadingView />,
+      (_, err) => ErrorView(err)
+    );
```

Comment:
What do you think about this little refactor? On the error branch why you don't use ErrorView defined above?

```suggestion
  const getDecodedPidOrError = (someDecodedPid: PidDecoded) => pipe(
          someDecodedPid,
          O.fold(
            () => <> </>, // TODO: https://pagopa.atlassian.net/browse/SIW-364
            decodedPid => <ContentView decodedPid={decodedPid} />
          )
        );
        
  const RenderMask = () =>
    pot.fold(
      decodedPid,
      () => <LoadingView />,
      () => <LoadingView />,
      () => <LoadingView />,
      err => ErrorView(err),
      getDecodedPidOrError,
      () => <LoadingView />,
      () => <LoadingView />,
      (_, err) => ErrorView(err)
    );
```

Reply:
I agree with the refactor, addressed in #33e63e37b881c1092de1ba78af0c8ea20d6b3024
However, I'll proceed with implementing a generic ErrorView first, which will have the capability to display a standardized view for various edge cases. Once that is done, I'll refactor the existing ErrorViews accordingly. Tracked via the mentioned SIW-364 issue.